### DISCUSSION TITLE
Feature Request - Ability to export or sync with google calendar or Ical

### DISCUSSION THREAD

**Discussion by lionslair** [bljohnsondev/mishos#8] [Ideas]
A useful feature or me would be the ability to provide an iCal feed or something so I can see the calendar in Google calendar and home assistant.

**Comment by bljohnsondev** [comment]
Interesting idea.  Since the API endpoints require authentication you would have to manually download the ical file to import to your calendar application.

I'm assuming you'd want the "Upcoming Shows" list in an ical format.

**Reply by Martin-Fox** [reply]
It will be nice feature if this calendar will update automatically (like I have with F1 calendar). But if I will have to manually sync my calendar... then no :)
